examples/framework-react/src/components/Counter.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 
-export default function Counter({ children }) {
-  const [count, setCount] = useState(0);
+export default function Counter({ children, count: initialCount }) {
+  const [count, setCount] = useState(initialCount);
```

Comment:
instead of adding this here, can you add as a test instead? These examples aren't tested so this doesn't guarantee that we won't break over time.

Reply:
Sure I'll write the UT.